src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(health = 100, name) {
```

Comment:
The constructor should accept 'name' as a parameter and set it accordingly. The 'health' parameter should have a default value of 100.